test/base/json.js

```diff
@@ -15,35 +15,74 @@ describe('JSON', () => {
     }
   ]);
 
-  let serialized = net.toJSON();
-  let net2 = new NeuralNetwork().fromJSON(serialized);
+  const serialized = originalNet.toJSON();
+  const serializedNet = new NeuralNetwork().fromJSON(serialized);
 
-  let input = {'0' : Math.random(), b: Math.random()};
-
-  it('toJSON()/fromJSON()', () => {
-    let output1 = net.run(input);
-    let output2 = net2.run(input);
-
-    assert.equal(JSON.stringify(output2), JSON.stringify(output1),
-      'loading json serialized network failed');
+  const input = {'0' : Math.random(), b: Math.random()};
+  describe('.toJSON()', () => {
+    describe('.layers', () => {
```

Comment:
Ticky-Tacky-Toey...I am not in love with the multiple describe lines, I don't mind them there but they kind of make the test output messy.  Look for comparison:

![image](https://user-images.githubusercontent.com/2294139/35459897-3dfe0fd8-029f-11e8-93cc-bb99812d7eb9.png)


I don't hate it enough to ask you to change just enough to complain about it a little. (who really says `Ticky-Tacky-Toey`

Reply:
lol
I don't prefer the output, but it is descriptive and allows us to compartmentalize the different items being exported both logically and for the human element.  I actually envisioned this expanding when more properties are potentially added here: https://github.com/BrainJS/brain.js/issues/111

Reply:
![energeticimpartialirrawaddydolphin-size_restricted](https://user-images.githubusercontent.com/2294139/35460746-45988fa4-02a2-11e8-8f45-4b63124bb3f0.gif)
![no-one-should-criticize-me](https://user-images.githubusercontent.com/2294139/35460749-472ffdb6-02a2-11e8-990c-fc9dd49fba77.gif)